### ISSUE TITLE
Rename to `babel-boilerplate`?

### ISSUE THREAD

**Issue by jmeas** [babel/babel-library-boilerplate#165]
Maybe? I first included the word `library` in the title to distinguish this from my boilerplate for authoring entire client-side web apps, but now they've converged to be the same thing.

**Comment by thejameskyle** [comment]
Idk, there's a number of other things to consider. I think people generally have different requirements for libraries vs. apps/sites.

**Comment by jmeas** [comment]
That's definitely true. For me, I've found that the difference is that I just need to add a little bit more to the boilerplate for webapps, which are mostly just webapp-specific opinions that are separate from Babel. I used to have an entirely different starting boilerplate for webapps, tho'.

I'm fine leaving it – I just thought I'd suggest the simpler name to see what people thought. So you think I should just keep it the same?
